arts-main/pubsub/handlers.py

```diff
@@ -135,11 +147,16 @@ def control(self, msg):
             str(msg.topic), msg.payload))
 
         if msg_type == 'runtime_resp':
-            return self.__create_module_ack(msg)
+            result = msg.get('data', 'result')
+            if(result == "no file"):
```

Comment:
This entire branch should go in __create_module_ack since it's more or less implementing the TODO.

Reply:
We send a module create message again. See previous comment for details.